### PR TITLE
Issue 66 - Implement image carousel on place details

### PR DESCRIPTION
**Pull request by fluffyemily**
# 66

### CODE REVIEW

**Comment by fluffyemily** [comment]
[I want to receive an actionable overview of relevant information about places near me, so that my day is not ruined with FOMO](https://trello.com/c/VPWm03rY/49-i-want-to-receive-an-actionable-overview-of-relevant-information-about-places-near-me-so-that-my-day-is-not-ruined-with-fomo)

File: Prox/Prox/PlaceDetails/PlaceDetailViewController.swift [hunk 1]
```diff
@@ -53,5 +96,53 @@ class PlaceDetailViewController: UIViewController {
...
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return place.photoURLs?.count ?? 1
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: CellReuseIdentifier, for: indexPath) as! ImageCarouselCollectionViewCell
+        if let photoURLs = place.photoURLs,
+            let photoURL = URL(string: photoURLs[indexPath.item]) {
+            cell.imageView.setImageWith(photoURL)
```
Comment (mcomella) [inline Prox/Prox/PlaceDetails/PlaceDetailViewController.swift]: I can't find this method in the docs – do you need to `git add` a file?

File: Prox/Prox/PlaceDetails/PlaceDetailViewController.swift [hunk 2]
```diff
@@ -53,5 +96,53 @@ class PlaceDetailViewController: UIViewController {
...
+
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+    }
+    
+}
+
+extension PlaceDetailViewController: UIScrollViewDelegate {
+    func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) {
+        let pageSize = imageCarousel.bounds.size
+        let selectedPageIndex = Int(floor((scrollView.contentOffset.x-pageSize.width/2)/pageSize.width))+1
```
Comment (mcomella) [inline Prox/Prox/PlaceDetails/PlaceDetailViewController.swift]: nit: spaces between math operators

File: Prox/Prox/PlaceDetails/PlaceDetailViewController.swift (hunk 1 above)
Comment (fluffyemily) [inline Prox/Prox/PlaceDetails/PlaceDetailViewController.swift]: it's in `UIImageView_AFNetworking` - an extension provided inside `AFNetworking`

**Pull request merged**